app/models/manageiq/providers/oracle/infra_manager.rb

```diff
@@ -0,0 +1,99 @@
+class ManageIQ::Providers::Oracle::InfraManager < ManageIQ::Providers::InfraManager
+  require_nested :RefreshWorker
+  require_nested :RefreshParser
+  require_nested :Host
+  require_nested :Refresher
+  require_nested :Template
+  require_nested :Vm
+
+  def self.ems_type
+    @ems_type ||= "oraclevm".freeze
+  end
+
+  def self.description
+    @description ||= "Oracle Virtualization Manager".freeze
+  end
+
+  def self.default_blacklisted_event_names
+    []
+  end
+
+  def supports_port?
+    true
+  end
+
+  def supported_auth_types
+    %w(default)
+  end
+
+  def supports_authentication?(authtype)
+    supported_auth_types.include?(authtype.to_s)
+  end
+
+  def self.raw_connect(server, port, username, password)
+    require 'fog/oracle'
+
+    url = "https://#{server}:#{port}"
+
+    params = {
+      :oracle_username => username,
+      :oracle_password => password,
+      :oracle_url      => url,
+      :connection_options => {
+        :ciphers => 'HIGH:RC4:!SSLv2:!aNULL:!eNULL:!3DES',
+        :ssl_verify_peer => false
+      }
+    }
+
+    Fog::Compute::Oracle.new(params)
+  end
+
+  def connect(options = {})
+    raise "no credentials defined" if self.missing_credentials?(options[:auth_type])
+
+    server   = options[:ip] || address
+    port     = options[:port] || self.port
+    username = options[:user] || authentication_userid(options[:auth_type])
+    password = options[:pass] || authentication_password(options[:auth_type])
+
+    self.class.raw_connect(server, port, username, password)
+  end
+
+  def oraclevm_service
+    @oraclevm_service ||= connect()
```

Comment:
@bancek that is comparing apples to oranges.  In the code I commented on, the `connect` method is being called without parameters, therefore no parens are needed.  In the code you referenced in your comment, `options` is an optional parameter to the method with a default value of `{}`